azure-managed-lustre/includes/client-install-ubuntu-22.md

```diff
@@ -9,6 +9,47 @@ ms.lastreviewed: 02/19/2023
 
 ---
 
+> [!IMPORTANT]
+> The Azure Marketplace image for the Ubuntu 22.04 LTS release uses the Hardware Enablement (HWE) Kernel by default. However, these kernels are only supported for 6 month periods, and Lustre support for these kernels is often not available when they are released. Therefore, it is recommended that you switch to the LTS kernel, as it will give you more stability as well as maintaining a kernel version that is supported with the Lustre Client.
+
+1. Install the LTS Kernel Metapackage.
+
+   ```bash
+   sudo apt update && sudo apt install linux-image-azure-lts-22.04
+   ```
+
+1. Remove the HWE Kernel Metapackage.
+
+   Remove the default (Hardware Enablement) Kernel metapackage, it will also ask to remove the linux-azure metapackage.  This is expected.
```

Comment:
```suggestion
   Remove the default (Hardware Enablement) Kernel metapackage. It will also ask to remove the linux-azure metapackage.  This is expected.
```